README.md

```diff
@@ -16,8 +16,8 @@ Fluent UI Apple contains native UIKit and AppKit controls aligned with [Microsof
 #### Requirements
 
 - iOS 13+ or macOS 10.14+
-- Xcode 11+
-- Swift 5.0+
+- Xcode 12.5+
+- Swift 5.4+
```

Comment:
The podspec is still on 5.0:

`  s.swift_version = "5.0"`

Reply:
Also, are there other places in the project where 5.0 is defined where we can bump it too?

Reply:
I think internally, swift version "5.x" is just read as "latest version of Swift 5" so it doesn't actually matter

Reply:
Hm well just to be safe I'll bump the podspec anyway.

There's one more place I can think of: `Package.swift` which currently names 5.3. I'll bump that to 5.4 as well.